### DISCUSSION TITLE
3D model without red and black borders

### DISCUSSION THREAD

**Discussion by Gleb-GG** [pyvista/pyvista#1672] [Q&A]
Hi pyvista team!

How is it possible to remove the red border and black box with add_mesh_slice (). That is, the 3D model should have the model itself in 3 slices without any borders

In fact, it requires visualization as in slice_orthogonal, but with interactivity

![image](https://user-images.githubusercontent.com/79473992/133369802-aa048cf0-f337-4360-b7e5-a6f89abb7363.png)

Thanks for your help

**Comment by banesullivan** [comment]
This is a known issue with the slicing widget. I tried searching for the previous discussion but cannot find it
